Earthfile

```diff
@@ -476,9 +478,25 @@ uki-iso:
     RUN xorriso -as mkisofs -V 'UKI_ISO_INSTALL' -e efiboot.img -no-emul-boot -o $ISO_NAME.iso /tmp/efi
     SAVE ARTIFACT /build/$ISO_NAME.iso kairos.iso AS LOCAL build/$ISO_NAME.uki.iso
 
+enki-image:
+    FROM  quay.io/kairos/enki:v0.0.8
+    SAVE ARTIFACT /enki enki
+
 # Uki stuff End
+uki-enki:
+        ARG --required BASE_IMAGE # BASE_IMAGE is existing kairos image which needs to be converted to uki
```

Comment:
not sure this is useful here. I mean, nice but will this build the current master kairos and use that as rootfs? Or this requires and already built and pushed image?

Reply:
it requires a built and pushed image ... the problem is earthly does not play well with local images. I think the only option is to use the `WITH DOCKER` but even that I tested and it seems to only support ubuntu/debian or something because it breaks with fedora.